definitions/divorce/json/AuthorisationCaseType.json

```diff
@@ -50,7 +50,7 @@
   {
     "LiveFrom": "16/05/2019",
     "CaseTypeID": "DIVORCE_NOTICE_OF_ACTING",
-    "UserRole": "caseworker-divorce-solicitor",
+    "UserRole": "caseworker-divorce-superuser",
```

Comment:
?

Reply:
@hosainnet did this change, was a bug fix I believe...

Reply:
this is the file 
``` {
    "LiveFrom": "16/05/2019",
    "CaseTypeID": "DIVORCE",
    "UserRole": "caseworker-divorce-superuser",
    "CRUD": "CRU"
  },
  {
    "LiveFrom": "16/05/2019",
    "CaseTypeID": "DIVORCE_NOTICE_OF_ACTING",
    "UserRole": "caseworker-divorce-superuser",
    "CRUD": "CRU"```

Reply:
https://github.com/hmcts/div-ccd-definitions/pull/153/commits/9f7bede2e57d0842360ac725590670b89ae7d224 yes it's meant to be

Reply:
ahh yup right